client/src/components/ElementEditor/Header.js

```diff
@@ -52,12 +67,23 @@ class Header extends Component {
       elementType,
       fontIcon,
       ActionMenuComponent,
+      expandable,
+      previewExpanded
     } = this.props;
 
     const deleteTitle = i18n._t('ElementHeader.DELETE', 'Delete');
 
     const deleteButtonClassNames = classNames('dropdown-item', 'element-editor__actions-delete');
 
+    const expandTitle = i18n._t('ElementHeader.EXPAND', 'Show editable fields');
+    const expandButtonClassNames = classNames(
+      'dropdown-item',
+      'element-editor__expand',
+      'btn',
+      { 'font-icon-right-open-big': !expandable,
+        'font-icon-up-open-big': expandable && previewExpanded,
+       'font-icon-down-open-big': expandable && !previewExpanded });
```

Comment:
Can you please format this object a little - each entry on a new line with consistent indentation